pkg/webui/console/components/device-map/index.js

```diff
@@ -0,0 +1,51 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import PropTypes from 'prop-types'
+import MapWidget from '../../../components/map/widget/'
+
+export default class DeviceMap extends React.Component {
+  render() {
+    const { device, ids } = this.props
```

Comment:
you only need the device here, right (assuming that the object has app/dev ids).
i dont see any reasons to pass those separately